### ISSUE TITLE
load pretrained ERROR

### ISSUE THREAD

**Issue by yangxiaoyanduter** [AlexeyAB/darknet#6691]

**Comment by yangxiaoyanduter** [comment]
hello ,thanks for viewing.  When i use pretrained weight to train my dataset,an error  occured .It requires me to train from the beginning ,i dont know how to deal 






Loading weights from yolov4.conv.137...
 seen 32, trained: 2150504 K-images (33601 Kilo-batches_64) 
Done! Loaded 137 layers from weights-file 
Learning Rate: 0.0005, Momentum: 0.949, Decay: 0.0005
 Detection layer: 139 - type = 28 
 Detection layer: 150 - type = 28 
 Detection layer: 161 - type = 28 
Saving weights to /home/yxy/darknet-master/backup/yolov4-aayang_final.weights
 Create 64 permanent cpu-threads 
If you want to train from the beginning, then use flag in the end of training command: -clear

**Comment by PriceZhang** [comment]
Hello, I also encountered this problem. I used yolov4.conv.137. Have you solved it?

**Comment by uuu686** [comment]
> 您好，我也遇到了这个问题。我使用yolov4.conv.137。解决了吗

Hello, I also encountered the same problem, do you know how to solve the problem?

**Comment by uuu686** [comment]
> Hello, I also encountered this problem. I used yolov4.conv.137. Have you solved it?

Hello, I also encountered the same problem, do you know how to solve the problem?
